tests/tpch/conftest.py

```diff
@@ -158,15 +158,15 @@ def cluster_spec(scale):
         }
     elif scale == 1000:
         return {
-            "worker_vm_types": ["m6i.large"],
+            "worker_vm_types": ["m6i.xlarge"],
```

Comment:
I noted that pyspark query 21 would always fail with this setup, but will succeed with m6i.2xlarge. 
Maybe that's okay though? If we succeed with this and pyspark mostly succeeds w/ other queries except one, then too bad for them(?).